src/engine/CodeValidator.v3

```diff
@@ -770,33 +770,33 @@ class CodeValidator(extensions: Extension.set, limits: Limits, module: Module, e
 				V128_LOAD_64_ZERO => checkLoad(opcode, 3, ValueType.V128);
 				V128_LOAD		=> checkLoad(opcode, 4, ValueType.V128);
 				V128_STORE		=> checkStore(opcode, 4, ValueType.V128);
-				V128_LOAD_8_LANE	=> { checkLaneLoad(opcode, 0); checkLane(15); }
-				V128_STORE_8_LANE	=> { checkLaneStore(opcode, 0); checkLane(15); }
-				V128_LOAD_16_LANE	=> { checkLaneLoad(opcode, 1); checkLane(7); }
-				V128_STORE_16_LANE	=> { checkLaneStore(opcode, 1); checkLane(7); }
-				V128_LOAD_32_LANE	=> { checkLaneLoad(opcode, 2); checkLane(3); }
-				V128_STORE_32_LANE	=> { checkLaneStore(opcode, 2); checkLane(3); }
-				V128_LOAD_64_LANE	=> { checkLaneLoad(opcode, 3);  checkLane(1); }
-				V128_STORE_64_LANE	=> { checkLaneStore(opcode, 3);  checkLane(1); }
+				V128_LOAD_8_LANE	=> { checkLaneLoad(opcode, 0); checkLane(opcode, 15); }
```

Comment:
Can you extract the validator changes from the change the implements new instructions? Also, can we name this `checkLaneAndSig` now?

Reply:
Sure. Pushed a new commit.